sandpack-react/src/components/CodeViewer/index.tsx

```diff
@@ -51,14 +51,24 @@ export const SandpackCodeViewer = React.forwardRef<
 
     const shouldShowTabs = showTabs ?? sandpack.openPaths.length > 1;
 
+    // decorators need to sort by `line`, otherwise it will throw error
+    // see https://github.com/codesandbox/sandpack/issues/383
+    const sortedDecorators = React.useMemo(
```

Comment:
Nice! 

Could you please do this logic at the CodeMirror component level? Or maybe in the`highlighDecorators.ts`? Then we make sure that we cover all possible components that use CodeMirror.

Reply:
Yeah sure, already updated, didn't drive deep into the code, so just chose the simplest way😂.